scripts/verify-ci-config.js

```diff
@@ -30,6 +45,9 @@ function checkYaml (yamlPath) {
     if (!job.env || !job.env.PLUGINS) continue
 
     const pluginName = job.env.PLUGINS
+    if (!yamlPath.includes('appsec')) {
```

Comment:
why not check appsec ones too ?

Reply:
That can probably be checked. I was concentrating on the issue at hand, which was non-appsec plugin tests.

Reply:
Actually, looking into it. `test:appsec:plugins` is somewhat non-trivial. Are you fine with this being handled in a subsequent PR?